src/containers/layout/layout.tsx

```diff
@@ -6,29 +6,28 @@ import { isVerifiedSession } from "~/utils/session.js";
 import { api } from "~/utils/api.js";
 import { fakeUsername } from "~/constants/fake-data/landing.js";
 import { useBBWallet } from "~/service/bb-wallet.js";
-import { UserStats } from "~/components/user-stats/index.js";
-
+import { Navbar } from "~/components/navbar/index.js";
 import { styles } from "./styles.js";
 
 export const Layout: FC<StackProps> = ({ children }) => {
   const { data: sessionData } = useSession();
   const isVerifiedUser = isVerifiedSession(sessionData);
-  const [open, setOpen] = useState(false);
+  const [menuIsOpen, setMenuIsOpen] = useState(false);
```

Comment:
```suggestion
  const [navbarIsOpen, setNavbarIsOpen] = useState(false);
```
since you renamed the component

Reply:
this actually tracks the main menu, the navbar is always visible so it doesn't have a "closed" state